python/cudf/udf_cpp/strings/include/cudf/strings/udf/udf_apis.hpp

```diff
@@ -50,30 +50,17 @@ class udf_string;
 std::unique_ptr<rmm::device_buffer> to_string_view_array(cudf::column_view const input);
 
 /**
- * @brief Return a STRINGS column given an array of udf_string objects
+ * @brief Return a STRINGS column given an array of managed_udf_string objects
```

Comment:
Does STRINGS in all caps have some special meaning? If not, let's use lowercase.
```suggestion
 * @brief Return a strings column given an array of managed_udf_string objects
```